src/navs/index.js

```diff
@@ -1762,6 +1762,16 @@ export const docsMenu = {
                                     name: 'Anonymous vs identified events',
                                     url: '/docs/data/anonymous-vs-identified-events',
                                 },
+                                {
+                                    name: 'Events Spec',
+                                    url: '/docs/data/event-spec/ecommerce-events',
+                                    children: [
+                                        {
+                                            name: 'Ecommerce Events',
+                                            url: '/docs/data/event-spec/ecommerce-events',
+                                        },
+                                    ],
+                                },
```

Comment:
I don't think we need to nest this until we have another event spec to add to the list. 
```suggestion
                                {
                                    name: 'Ecommerce events spec',
                                    url: '/docs/data/event-spec/ecommerce-events',
                                },
```